### PR TITLE
Python timeout

### PR DESCRIPTION
**Pull request by oleurud**
When our database timeout limits are exceeded we usually get a proper error message, but if the timeout is hit while the database is executing python code, the error message and status code are not correct.

```
{"error":["RuntimeError: Execution of function interrupted by signal"]}
```

Thanks @jgoizueta for the alert

### CODE REVIEW

**Comment by oleurud** [comment]
Looking the problem with the tests in Travis again, I have found something. Before running the tests, after the system configuration, we run this command:
`psql -A -t -c 'select version()'`

The result should be
`PostgreSQL 9.5.2 on ...`

but instead of it, we have:
`PostgreSQL 9.5.7 on ...`

So Travis is running a different version which has not the plpython fix

**Comment by oleurud** [comment]
It is blocked by https://github.com/CartoDB/CartoDB-SQL-API/pull/455

File: app/postgresql/error_handler.js [hunk 1]
```diff
@@ -16,7 +16,10 @@ ErrorHandler.prototype.getMessage = function() {
     // `57014: query_canceled` includes other queries than `statement timeout`, otherwise we could do something more
     // straightforward like:
     // return conditionToMessage[this.err.code] || this.err.message;
-    if (message && message.match(/statement timeout/)) {
+    if (message && 
+        (message.match(/statement timeout/) ||
```
Comment (dgaubert) [inline app/postgresql/error_handler.js]: I'd extract `(message.match(/statement timeout/) || message.indexOf('RuntimeError: Execution of function interrupted by signal') > -1)` condition to a function properly named.

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,2272 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (dgaubert) [inline yarn.lock]: Remove this file. This project doesn't require yarn to be installed.

File: app/postgresql/error_handler.js (hunk 1 above)
Comment (dgaubert) [reply app/postgresql/error_handler.js]: In fact, the whole condition. ;)

File: yarn.lock (hunk 2 above)
Comment (oleurud) [reply yarn.lock]: Thanks :fearful:

**Comment by oleurud** [comment]
[Python timeout error](https://trello.com/c/Scz2Svn9/2-python-timeout-error)

**Pull request merged**